src/_includes/cloud/note-prevent-site-availability-issues.md

```diff
@@ -0,0 +1,2 @@
+ {:.bs-callout-warning}
+This operation triggers the Cloud build and deploy process, which takes your site offline until deployment completes. For Staging and Production environments, we recommend completing this operation during off-peak hours to prevent service disruptions.
```

Comment:
```suggestion
This operation triggers the Cloud build and deploy process, which takes your site offline until deployment completes. For Production environments, we recommend completing this operation during off-peak hours to prevent service disruptions.
```